helper/staking/staking.go

```diff
@@ -138,10 +143,17 @@ func PredeployStakingSC(
 		return nil, fmt.Errorf("unable to generate DefaultStatkedBalance, %w", err)
 	}
 
+	if maxValidatorCount > math.MaxUint32 {
+		maxValidatorCount = math.MaxUint32
+	} else if maxValidatorCount < 4 {
+		return nil, fmt.Errorf("maximum number of validator can not be lower than 4")
+	}
```

Comment:
This check shouldn't be here, as there shouldn't be a minimum number of nodes you need to form a PoS cluster, apart from 1.

So I suggest setting this check to `1` (and of course extracting it into a constant, we don't want magic numbers 🙂 )